lightningd/plugin.c

```diff
@@ -23,8 +26,27 @@ struct plugin {
 	const char *outbuf;
 };
 
+struct plugin_request {
+	u64 id;
+	/* Method to be called */
+	const char *method;
+
+	/* JSON encoded params, either a dict or an array */
+	const char *json_params;
+	const char *response;
+	const jsmntok_t *resulttok, *errortok;
+
+	/* The response handler to be called on success or error */
+	void (*cb)(const struct plugin_request *, void *);
+	void *arg;
+};
+
 struct plugins {
 	struct plugin **plugins;
+	int pending_init;
```

Comment:
size_t, since it's a count of things.  `int`is used by old-school C programmers in place of `bool`, so I first wondered if this was a boolean...